pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -124,7 +129,16 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 			plugins = append(plugins, saAdmitter)
 
 		default:
-			plugin := admission.InitPlugin(pluginName, kubeClient, server.AdmissionControlConfigFile)
+			configFile := server.AdmissionControlConfigFile
+			pluginConfig := options.KubernetesMasterConfig.AdmissionConfig.PluginConfig
+
+			if cfg, hasConfig := pluginConfig[pluginName]; hasConfig {
```

Comment:
you don't check for existence in origin.  Is this an unnecessary check?  I suspect so.